src/github/workflows/test.yml

```diff
@@ -85,6 +85,8 @@ jobs:
     runs-on: &runs_on ubuntu-22.04
     steps:
     - uses: styfle/cancel-workflow-action@0.11.0
+      # Andreas, 2022-12-05, issue #6388: do not cancel on `master`!
+      if: github.ref != 'refs/heads/master'
```

Comment:
PR updates do not cancel the workflow anymore, so it seems that for PR updates the ref is also `master` if the PR is against `master` !?